eo-maven-plugin/src/main/java/org/eolang/maven/PullMojo.java

```diff
@@ -103,7 +106,7 @@ public void exec() throws IOException {
                 new OyCaching(
                     this.hash,
                     this.outputPath,
-                    new OyRemote(this.hash)
+                    new OyRemote().init(this.hash, maxtries, timeout)
```

Comment:
@yasamprom Maybe it can be `new OyRemote(this.hash).resolve()`? 

Also I think `PullMojo` don't need to know about `OyCaching`'s constants. But it is up to you.